### PR TITLE
Families now get thematic weapons

### PR DESCRIPTION
**Pull request by Shadowflame909**
## About The Pull Request
Since families are becoming more intertwined within /tg/ and will join the likes of dynamic. I gave the gangsters thematic weapons to even the score for when they eventually interact with other antags.

GTA Gangsters: They get some unused guns that correlate with the weapons the specific gangsters actually used. (Did some research on the GTA wiki) Like an uzi, SMG, assault rifle, and a pump shotgun. EX: Mafia gets a tommy gun, Yakuza gets a katana.

Jack Frost Gang: Temp gun, the cold genetic powers, and a gluon grenade.

Dutch Gang: This one I did some digging into, and since the dutch come from the Netherlands. And it's LITERALLY illegal over there to have any sort of weaponry, even pepper spray will get you in trouble. They're doing with a metal ablative bat since the dutch don't do actual weaponry.

The Monarch: Butterfly themed, they get the closest thing we have, bee-themed weapons. A beesaber and a bee grenade.

## Why It's Good For The Game
I've always felt that these antags were lacking in anything to differentiate themselves from the average greytide. All they had were matching jumpsuits and spray-cans, and that's it. With the families restraint on killing outside of gang warfare you had the perfect combination for average greytiding, even more, lackluster than a revolutionary. 

Now that they're being involved in the dynamic mixup. They have nothing to even the playing field when every other antag will have super weapons like dual e-swords, arm blades, lg-saws, an eldritch abomination as a backup, you get the point. These guys would get washed and ultimately feel like a wasted antag roll. Since you get nothing but a jumpsuit.

Now with thematic weaponry, they're gonna be able to first off equally be able to defend themselves should any other antag start preying on them with things only unique to them. Even better now, they're going to be able to bring some unique violence upon each other with their own thematic weaponry.

## Changelog
:cl:
balance:  The Families now have upgraded their arsenal with thematic weapons, some of them even counter each other.

/:cl:

### CODE REVIEW

**Comment by Shadowflame909** [comment]
The PR that sparked my desire to give families something other than a jumpsuit and a spraycan: #51244
Now here's some quick FAQ.

**Q: Won't it be too OP to give families top tier weapons such as these shadowflame909? Isn't that why gangs was so hated**

A: Families so far is very dissuaded from causing unreasonable levels of violence, with the star system and the flavor text. These weapons serve two purposes, protection from other antags, and more interesting battles with other gangsters.

Ex: The Monarch gang versus the Jack Frost Gang. Gluon Grenade kills the bees from temp damage. Monarch gang has to close in on them with beesabers, while the jack frost gang is chucking snowballs and building snow-walls for defense.

Ex 2: The Dutch Gang with their reflective bats versus the Yakuza with their blocking swords. Have them both find lasers from somewhere, and now you have an interesting match of pong.

File: code/modules/antagonists/gang/gang.dm [hunk 1]
```diff
@@ -354,7 +369,8 @@
 	free_clothes = list(/obj/item/clothing/under/costume/dutch,
 						/obj/item/clothing/head/bowler,
 						/obj/item/clothing/suit/dutch,
-						/obj/item/toy/crayon/spraycan)
+						/obj/item/toy/crayon/spraycan
+						/obj/item/melee/baseball_bat/ablative)
```
Comment (necromanceranne) [inline code/modules/antagonists/gang/gang.dm:373]: Are you actually being serious

**Comment by Iamgoofball** [comment]
:-1:

If you absolutely want to give gangs thematic weapons, they need to be set up like this:
1. Melee weapon.
2. One handed.
3. Blunt, not sharp.
4. 15 damage, brute.
5. Given upon sign-up for the gang.
6. Unique sprites.

File: code/modules/antagonists/gang/gang.dm (hunk 1 above)
Comment (Shadowflame909) [reply code/modules/antagonists/gang/gang.dm:373]: Yeah look at my in-depth reason and the faq discussion of how I preview it playing out.

**Comment by necromanceranne** [comment]
There isn't a thing here that isn't utterly terrible, and this is a terrible idea. I feel like this is immensely bad faith and some of these aren't even remotely thematic, you're just givign them the strongest wea[pm you can fit into the slot. Some of these are laughably out of touch with each other. The Vagos get a plink rifle for fucks sake, what did they do to deserve having a WT550 shoved into their hands when the mafia get a fucking tommy gun.

I'd go with goofs suggestion.

File: code/modules/antagonists/gang/gang.dm (hunk 1 above)
Comment (necromanceranne) [reply code/modules/antagonists/gang/gang.dm:373]: The Dutch gang are cowboys. And katanas qdel projectiles they block. Even in the absolute lunatic internal logic you got, it's incorrect.

**Pull request closed without merging**